Makefile

```diff
@@ -24,37 +26,58 @@ sedseparator = ''
 sedignorecase =
 endif
 
-build_dir = 'tmp/_output'
+build_dir := build/_output
 configcli_dest := $(build_dir)/configcli.tgz
 
 .DEFAULT_GOAL := build
 
 build: configcli pkg/apis/kubedirector.bluedata.io/v1alpha1/zz_generated.deepcopy.go | $(build_dir)
-	@echo
 	@echo \* Creating KubeDirector deployment image and YAML...
 	@test -d vendor || dep ensure -v
-	@echo operator-sdk build ${image}
-	@operator-sdk build ${image} | grep -v "Create deploy/operator.yaml"
+	operator-sdk build ${image}
 	@docker image prune -f > /dev/null
-	@sed -i ${sedseparator} \
-        -e '/command:/ {' \
-        -e 'n; ' \
-        -e 's~.*~          - "/bin/sh"~; G; ' \
-        -e 's~$$~          args:~; G; ' \
-        -e 's~$$~          - "-c"~; G; ' \
-        -e 's~$$~          - "mkfifo /tmp/fifo; (/root/${project_name} \&> /tmp/fifo) \& while true; do cat /tmp/fifo; done"~;' \
-        -e '}' deploy/operator.yaml
-	@sed -i ${sedseparator} \
+	@sed ${sedseparator} \
+        -e '/command:/ { n; ' \
+        -e 's~.*~          - "/bin/sh"~; ' \
+        -e 'a\          args:' \
+        -e 'a\          - "-c"' \
+        -e 'a\          - "mkfifo /tmp/fifo; (/root/${project_name} \&> /tmp/fifo) \& while true; do cat /tmp/fifo; done"' \
+        -e '}' \
         -e '/env:/ {' \
-        -e 'G; ' \
-        -e 's~$$~            - name: MY_NAMESPACE~; G; ' \
-        -e 's~$$~              valueFrom:~; G; ' \
-        -e 's~$$~                fieldRef:~; G; ' \
-        -e 's~$$~                  fieldPath: metadata.namespace~;' \
-        -e '}' deploy/operator.yaml
+        -e 'a\            - name: MY_NAMESPACE' \
+        -e 'a\              valueFrom:' \
+        -e 'a\                fieldRef:' \
+        -e 'a\                  fieldPath: metadata.namespace' \
+        -e '}' \
+        -e 's~REPLACE_IMAGE~${image}~' \
```

Comment:
Now that operator.yaml is checked into our repo rather than being generated, it seems like we can avoid most of this sed stuff (by having the desired text just already be in operator.yaml). Maybe the only thing we still need to change in operator.yaml at buildtime is REPLACE_IMAGE.

Reply:
Ok, I modified the operator.yaml and removed most of the sed stuff, except for the REPLACE_IMAGE line, as you suggested.

Reply:
There is nothing in our Makefile that invokes build.sh or docker_build.sh so I'm assuming it's the operator-sdk build command. I moved the scripts to build/bin and the build still works.